website/static/install

```diff
@@ -0,0 +1,135 @@
+#!/bin/sh
+
+set -eu
+
+INSTALLPATH=/usr/local/bin
+
+happyexit() {
+  echo "Installed version:"
+  echo "$(talosctl version 2>/dev/null )"
+  echo ""
+  echo "Now run:"
+  echo ""
+  echo "  talosctl cluster create                  # install a local test cluster"
+  echo "  talosctl dashboard                       # If you have created a cluster, launch the dashboard"
+  echo ""
+  echo "Looking for more? Visit https://talos.dev/latest"
+  echo ""
+  exit 0
+}
+
+validate_checksum() {
+  filename=$1
+  url="https://github.com/siderolabs/talos/releases/latest/download/sha256sum.txt"
+  SHA=$(curl --proto '=https' --tlsv1.2 -sSfL "${url}" | grep "${filename}" | awk '{print $1 }')
+  echo ""
+  echo "Validating checksum..."
+
+  case $checksumbin in
+    *openssl)
+      checksum=$($checksumbin dgst -sha256 "${filename}" | sed -e 's/^.* //')
+      ;;
+    *shasum)
+      checksum=$($checksumbin -a256 "${filename}" | sed -e 's/ .*$//')
+      ;;
+  esac
+
+  if [ "$checksum" != "$SHA" ]; then
+    echo "Checksum validation failed." >&2
+    return 1
+  fi
+  echo "Checksum valid."
+  return 0
+}
+
+OS=$(uname -s)
+arch=$(uname -m)
+cli_arch=""
+case $OS in
+  CYGWIN* | MINGW64*)
+    OS=windows-amd64.exe
+    ;;
+  Darwin | Linux | FreeBSD)
```

Comment:
I don't remember how WSL 2 reports itself, but it'd be nice to have it detected as Linux